config/application.rb

```diff
@@ -21,7 +21,7 @@ class Application < Rails::Application
     #
     # config.time_zone = "Central Time (US & Canada)"
     # config.eager_load_paths << Rails.root.join("extras")
-    config.i18n.load_path += Dir[Rails.root.join("config", "locales", "**",
-                                                 "*.{rb,yml}").to_s]
+    # config.i18n.load_path += Dir[Rails.root.join("config", "locales", "**",
```

Comment:
Cái ni mình không nên comment lại nè, để cái ni để cái locales I18n nó load được hết file trong folder locales như ở [đây](https://guides.rubyonrails.org/i18n.html#organization-of-locale-files)